_rules/SC4-1-2-iframe-unique-name.md

```diff
@@ -0,0 +1,136 @@
+---
+name: IFrame has a unique accessible name
+
+description: |
+ Each iframe element has a unique accessible name
+
+success_criterion:
+- 4.1.2
+
+test_aspects:
+- DOM Tree
+- CSS Styling
+
+authors:
+- Jey Nandakumar
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to `iframe` elements within a given [document context](#document-context) that have either `title`, `aria-label` or `aria-labelled-by` attribute.
```

Comment:
I don't quite know where this document context definition lives. I take it you want this to ignore shared names between frames? Does this also apply to stuff in shadow DOM?